### PR TITLE
Version 0.3

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#18 
#### What does this implement/fix? Explain your changes.
Version 0.3 with the full support of scikit Decision Tree models prepared.

### CODE REVIEW

**Comment by AHReccese** [comment]
@sepandhaghighi 
Hi King Sepand, I hope you're feeling better today.
It is a tiny minor PR that mainly changes versions from `0.2` to `0.3`.
I suggest you kindly review this minor PR til 9/27/2023.

Thanks, AHR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
```
Comment (sepandhaghighi) [inline CHANGELOG.md:8]: Please remove this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### Changed
```
Comment (sepandhaghighi) [inline CHANGELOG.md:9]: Please remove this line too

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AHReccese) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (AHReccese) [reply CHANGELOG.md]: Done.

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**